src/constants/placeholder.js

```diff
@@ -1 +1,42 @@
 // Some placeholder file so we may commit this folder to repo
+
+export const SIZES = [
+  'N/A',
+  'OS',
+  'XS',
+  'Small',
```

Comment:
We can place this in a dedicated file of it's own where the items in this constant file will contain values that are relevant to each other.